spec/models/event_spec.rb

```diff
@@ -188,4 +188,45 @@
     event.lock_application_status
     expect(event.application_status_locked).to eq(true)
   end
+
+  it "is in draft phase" do
+    event = FactoryGirl.build(:event)
+    event.draft = true
+    expect(event.phase).to eq(:draft)
+  end
+
+  it "is in application phase" do
+    event = FactoryGirl.build(:event)
+    event.draft = false
```

Comment:
True, but I dont think its good to rely on the default values in factories, since they might be changed